packages/core/src/conflict-resolution.ts

```diff
@@ -99,10 +98,14 @@ export async function pickLogToAccept(
       )
     }
 
-    // Compare block heights to decide which to take
-    if (proof1.blockNumber < proof2.blockNumber) {
+    // Compare anchor block timestamp to decide which to take.  Even though blockTimestamps can
+    // drift from wallclock time, they are guaranteed to be increasing in increasing blockNumbers,
+    // so they are still safe to compare for relative ordering.
```

Comment:
Is this enforced by ethereums consensus logic? Maybe add a link to some reference here?